test/load/terraform/test-servers.tf

```diff
@@ -26,37 +26,24 @@ resource "aws_security_group" "test-servers" {
   }
 }
 
-resource "aws_launch_configuration" "test-servers" {
-  name_prefix     = "${var.cluster_name}-${local.random_name}-test-"
-  image_id        = var.test_server_ami
-  instance_type   = var.test_instance_type
-  key_name        = module.keys.key_name
-  security_groups = [aws_security_group.test-servers.id]
-
+resource "aws_instance" "test-server" {
+  ami                         = var.test_server_ami
```

Comment:
Should this use a data source to pull the latest `consul-test-*` image if `test_server_ami` is not defined? That way if I don't need to modify the test server at all (which is probably rare) I can just use whatever is in the AWS account and not have to go and figure out what that AMI ID is and pass it to TF.